helpers/integration_test.go

```diff
@@ -0,0 +1,230 @@
+package helpers_test
+
+import (
+	"time"
+
+	"github.com/go-logr/logr"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	tektonv1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	applicationapiv1alpha1 "github.com/redhat-appstudio/application-api/api/v1alpha1"
+	"github.com/redhat-appstudio/integration-service/gitops"
+	"github.com/redhat-appstudio/integration-service/helpers"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
The staticcheck-action is complaining about the `v1 "k8s.io/apimachinery/pkg/apis/meta/v1` being imported twice here: https://github.com/redhat-appstudio/integration-service/actions/runs/3304067425/jobs/5452735358

Reply:
Done.

Reply:
Thanks!